core/src/test/scala/io/finch/EntityEndpointLaws.scala

```diff
@@ -12,13 +12,12 @@ trait EntityEndpointLaws[A] extends Laws with MissingInstances with AllInstances
 
   def decoder: DecodeEntity[A]
   def classTag: ClassTag[A]
-  def endpoint: Endpoint[Option[String]]
-  def serialize: String => Input
+  def endpoint: Endpoint[Option[A]]
```

Comment:
Maybe we can keep the tests unchanged for now?

Reply:
I think it should be ok to slightly change it toward new syntax